README.md

```diff
@@ -187,29 +187,41 @@ Need a bit of security? Use SSL sockets:
 ```
 $ puma -b 'ssl://127.0.0.1:9292?key=path_to_key&cert=path_to_cert'
 ```
-#### Self-signed SSL certificates (via _localhost_ gem, for development use): 
+#### Self-signed SSL certificates (via the [`localhost`] gem, for development use): 
 
-Puma supports [localhost](https://github.com/socketry/localhost) gem for self-signed certificates. This is particularly useful if you want to use Puma with SSL locally, and self-signed certificates will work for your use-case. Currently, `localhost-authority` can be used only in MRI. 
+Puma supports the [`localhost`] gem for self-signed certificates. This is particularly useful if you want to use Puma with SSL locally, and self-signed certificates will work for your use-case. Currently, the integration can only be used in MRI. 
 
-To use [localhost](https://github.com/socketry/localhost), you have to `require "localhost/authority"`: 
+Puma will automatically configure SSL if you require the [`localhost`] gem when running in environment `development`:
 
 ```ruby
-# Easiest way, in your Gemfile:
+# Add the localhost gem to your Gemfile
 group(:development) do 
-  gem 'localhost', require: 'localhost/authority'
-end 
+  gem 'localhost'
+end
+
+# config.ru:
```

Comment:
I was pretty confused by this, maybe it would make more sense as a sentence?

```suggestion
# Alternatively, you can require the gem in config.ru:
```